src/config/index.ts

```diff
@@ -28,3 +28,4 @@ export const LOTTERY_MAX_NUMBER_OF_TICKETS = 50
 export const LOTTERY_TICKET_PRICE = 1
 export const DEFAULT_TOKEN_DECIMAL = BIG_TEN.pow(18)
 export const DEFAULT_GAS_LIMIT = 200000
+export const DEFAULT_GAS_PRICE = '5000000000'
```

Comment:
```suggestion
export const DEFAULT_GAS_PRICE = '5000000000'

export const DEFAULT_GAS_PRICE = parseUnits('5', 'gwei')
```

Readability would be enhanced using utils from `ethers` to format gasPrice directly

Reply:
A note on this - I tried using `parseUnits` and Typescript started reporting a bunch of errors. I wonder if there is a version mismatch or ethers does not support a certain version of Bignumber.